src/core/components/live-response.jsx

```diff
@@ -37,6 +38,7 @@ export default class LiveResponse extends React.Component {
     const returnObject = headersKeys.map(key => {
       return <span className="headerline" key={key}> {key}: {headers[key]} </span>
     })
+    const hasHeaders = returnObject.length !== 0
```

Comment:
Can we make this `returnObject > 0`? I'm concerned about a possible edge case where `returnObject` is null or undefined.